Tests/YCoreUITests/Extensions/UIKit/UIView+constrainSizeTests.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  UIView+constrainSizeTests.swift
+//  YCoreUITests
+//
+//  Created by Dharmik Ghelani on 17/10/22.
+//  Copyright © 2022 Y Media Labs. All rights reserved.
+//
+
+import XCTest
+@testable import YCoreUI
+
+final class UIViewConstrainSizeTests: XCTestCase {
+    
```

Comment:
This whitespace causes a SwiftLint violation
```suggestion
```

Reply:
Fixed